### PR TITLE
Add Commands

### PR DESCRIPTION
**Pull request by daggy1234**
Yes

### CODE REVIEW

**Comment by daggy1234** [comment]
Tested on both mine and pikaninja's machine. Should work great. Nice autocomplete and stuff if I do say so myself

File: index.js [hunk 1]
```diff
@@ -40,18 +40,89 @@ module.exports = class UnblockNSFW extends Plugin {
...
+                    commands: [
+                        {
+                            command: "enable",
+                            description: "Enable NSFW"
+                        },
+                        {
+                            command: "disable",
+                            description: "disable NSFW"
+                        }],
+                    header: "neko subcommands",
```
Comment (SebbyLaw) [inline index.js]: what?

**Comment by SebbyLaw** [comment]
Merged in 3bd260cdfd2e600fe4c4e602dc396fc08d1a67f3

**Comment by daggy1234** [comment]
why not just merge :hmm:

**Pull request closed without merging**